src/dataflow/onmt_helpers/create_onmt_prediction_report.py

```diff
@@ -81,6 +85,14 @@ def is_correct_leaderboard(self) -> bool:
             and self.program_execution_oracle.refer_are_correct
         )
 
+    @property
+    def is_correct_leaderboard_ignoring_refer(self) -> bool:
+        """Returns true if the gold and the prediction match after canonicalization, ignoring the `refer_are_correct`.
+
+        Use this metric is you only care about the accuracy of the semantic parser, ignoring the execution of `refer`.
+        """
+        return self.gold_canonical == self.prediction_canonical
```

Comment:
:sheep: can you make line 84 be is_correct_leaderboard_ignoring_refer(self) and self.program_execution_oracle.refer_are_correct